.github/pull_request_template.md

```diff
@@ -9,7 +9,3 @@ Pre-checks:
 - [ ] Files has been included where required (if applicable)
 - [ ] Files removed have been deleted, not just excluded from the build (if applicable)
 - [ ] (New product features only) Release Note added
-
-Review:
-- [ ] Documentation Team Lead has reviewed
```

Comment:
Should this have been removed? I thought we only wanted the PM one removed, as we still need Brian to make sure PRs are reviewed before they are merged.

Reply:
Oh, sorry, I misunderstood. Can we make him a required reviewer instead, as we've done with the review group on the TinyMCE PRs?

Reply:
Not easily, because it has the issue of only having Brian in the reviewers list meaning no one else would get added unless people remember, which from the TinyMCE side of things seems to be forgotten **a lot.**

Reply:
ok. I'll put it back.